### PR TITLE
Adding personal information for reviewers into database

### PR DESCRIPTION
**Pull request by shreyabarua**
Reviewers' name and email are saved in Datastore. Can now be retrieved to personalize resume-review comments.

### CODE REVIEW

File: resume-buddy/src/main/java/com/google/sps/servlets/ReviewerDataServlet.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    reviewerEntity.setProperty("last-name", lname);
+    reviewerEntity.setProperty("email", email);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(reviewerEntity);
+  }
+
+  /**
+   * @return the request parameter, or the default value if the parameter was not specified by the
+   *     client
```
Comment (oydang) [inline resume-buddy/src/main/java/com/google/sps/servlets/ReviewerDataServlet.java]: Nit: weird spacing before client

File: resume-buddy/src/main/java/com/google/sps/servlets/ReviewerDataServlet.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that saves reviewer data from the form */
+@WebServlet("/reviewer-data")
+public class ReviewerDataServlet extends HttpServlet {
+
+  private Reviewer reviewer;
```
Comment (oydang) [inline resume-buddy/src/main/java/com/google/sps/servlets/ReviewerDataServlet.java:18]: This `Reviewer` isn't really being used right now at all. Remove it for now or add a TODO above it?

**Review by oydang — CHANGES_REQUESTED** [review]

File: resume-buddy/src/main/java/com/google/sps/servlets/ReviewerDataServlet.java (hunk 2 above)
Comment (shreyabarua) [reply resume-buddy/src/main/java/com/google/sps/servlets/ReviewerDataServlet.java:18]: I use reviewer to create the Gson/json in doGet()

Comment (oydang) [reply resume-buddy/src/main/java/com/google/sps/servlets/ReviewerDataServlet.java:18]: Oh I see it now! Sorry for not seeing it earlier.

**Review by oydang — APPROVED** [review]

**Pull request merged**